css/style.css

```diff
@@ -0,0 +1,308 @@
+*{
+    box-sizing: border-box;
+    font-family: 'Roboto', sans-serif;
+}
+a:link,
+a:visited,
+a:active,
+a:hover{
+	text-decoration: none;
+}
+iframe{
+	width: 100%;
+}
+img{
+	max-width: 100%;
+}
+body{
+	margin: 0;
+	padding: 0;
+}
+header{
+	padding: 0 16px;
+	height: 56px;
+	border-bottom: 1px solid #f0f0f0;
+	display: flex;
+	    justify-content: space-between;
+	background: rgb(252, 252, 252);
+}
+#left-header{
+	display: flex;
+}
+#main-nav{
+	background: transparent;
+	border: 0;
+	font-size:24px;
+	margin-right: 16px;
+}
+#logo{
+	max-height: 100%;
+}
+#search-form-container{
+	position: relative;
+	padding-right: 64px;
+}
+#search-form-container input[type="text"]{
+	width: 562px;
+  padding: 5px;
+}
+#search-form-container input[type="text"],
+#search-form-container button{
+	height: 32px;
+	margin-top: 12px;
+}
+#search-form-container button{
+	width: 65px;
+	background: #f0f0f0;
+	border: 1px solid #d3d3d3;
+	position: absolute;
+	top: 0;
+	right: 0;
+  color: rgb(96, 96, 96);
+}
+#options-container{
+	display: flex;
+}
+#options-container button{
+	margin-left: 22px;
+  color: rgb(96, 96, 96);
+}
+#cam,
+#grid-icon,
+#comments-icon,
+#user-icon,
+#bill-icon{
+	width: 24px;
+	height: 24px;
+	background: transparent;
+	border: 0;
+	padding: 0;
+	margin-top: 16px;
+  cursor: pointer;
+}
+#grid-icon{
+	background: url(../imgs/icons.png);
+	background-position: 5px 5px;
+	background-repeat: no-repeat;
+}
+#comments-icon,
+#bill-icon{
+    font-size: 18px;
+
+}
+
+#user-icon {
+  background-color: red;
+  color: #fff!important;
+  border-radius: 50%;
+}
+
+.container {
+  padding: 30px;
+}
+
+.video-title {
+  font-size: 22px;
+  font-weight: normal;
+  padding: 2px 0;
+}
+
+main {
+  float: left;
+  width: 73%
+}
+aside {
+  float: right;
+  width: 25%;
+}
+.pull-left{
+	float: left;
+}
+.pull-right{
+	float: right;
+
+}
+.row{
+	width: 100%;
+}
+.row:after{
+	content: "";
+	display: block;
+	clear:both;
+}
+.stats-left,
+.stats-right{
+	width: 50%;
+}
+.stats-left{
+	line-height: 54px;
+}
+.stats-list{
+	position: relative;
+	list-style-type: none;
+	display: flex;
+	/*flex-direction: row-reverse;*/
+    justify-content: space-between;
+    font-size: 20px;
+    color: #909090;
+    margin-bottom: 0;
+}
+.stats-list a{
+	color: #909090;
+}
+.stats-list li{
+	display: flex;
+}
+.stats-list li span{
+	padding-left: 8px;
+	font-size: 13px;
+	line-height: 20px;
+	/*padding-top: 2px;*/
```

Comment:
Remove this line as well. Always leave your CSS code free of unneeded styling comments.